CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingDataSource.swift

```diff
@@ -20,93 +19,81 @@ protocol OnboardingScreen: AnyObject {
-        let item = itemSections[indexPath.row]
-        let image = UIImageNamedPreferred(named: item.imageName)
-        guard let onboardingPageType = OnboadingPageType.init(rawValue: indexPath.row) else {
-            return
-        }
-        if let adapter = adapters[onboardingPageType], adapter != nil {
-            cell.iconView.illustrationAdapter = adapter
+        let pageModel = pages[indexPath.row]
+
+        if giniConfiguration.customOnboardingPages == nil {
+            if let adapter = pageModel.illustrationAdapter {
+                cell.iconView.illustrationAdapter = adapter
+                cell.iconView.setupView()
+            } else {
+                cell.iconView.icon = UIImageNamedPreferred(named: pageModel.page.imageName)
+            }
         } else {
-            cell.iconView.illustrationAdapter = ImageOnboardingIllustrationAdapter()
-            cell.iconView.icon = image
+            cell.iconView.icon = UIImageNamedPreferred(named: pageModel.page.imageName)
         }
-        cell.iconView.accessibilityValue = item.title
-        cell.iconView.setupView()
+        cell.iconView.accessibilityValue = pageModel.page.title
 
-        cell.descriptionLabel.text = item.description
-        cell.descriptionLabel.accessibilityValue = item.description
-        cell.titleLabel.text = item.title
-        cell.titleLabel.accessibilityValue = item.title
+        cell.descriptionLabel.text = pageModel.page.description
+        cell.descriptionLabel.accessibilityValue = pageModel.page.description
+        cell.titleLabel.text = pageModel.page.title
+        cell.titleLabel.accessibilityValue = pageModel.page.title
+    }
+
+    private func defaultOnboardingPagesDataSource() -> [OnboardingPageModel] {
+        var pageModels = [OnboardingPageModel]()
+
+        let flatPaperPageModel = (page: OnboardingPage(imageName: DefaultOnboardingPage.flatPaper.imageName,
+                                                       title: DefaultOnboardingPage.flatPaper.title,
+                                                       description: DefaultOnboardingPage.flatPaper.description),
+                                  illustrationAdapter: giniConfiguration.onboardingAlignCornersIllustrationAdapter)
+
+        let goodLightingPageModel = (page: OnboardingPage(imageName: DefaultOnboardingPage.lighting.imageName,
+                                                          title: DefaultOnboardingPage.lighting.title,
+                                                          description: DefaultOnboardingPage.lighting.description),
+                                     illustrationAdapter: giniConfiguration.onboardingLightingIllustrationAdapter)
+
+        pageModels = [flatPaperPageModel, goodLightingPageModel]
+
+        if giniConfiguration.multipageEnabled {
+            pageModels.append((page: OnboardingPage(imageName: DefaultOnboardingPage.multipage.imageName,
```

Comment:
[Suggestion]: Could you please extract the model for Multi-page into a variable?